src/shared/components/palette/utils/date.pipe.ts

```diff
@@ -40,7 +40,8 @@ export class DatePipe implements PipeTransform {
         if (this.formatTrans && format && format !== 'short') {
           // support for java style formatting strings for dates
           format = this.translateDateFormat(format);
-          resultDate = formatDate(date, format, 'en-GB', zone)
+          const thisMoment = moment(date).format(format);
```

Comment:
_Nitpick:_
Somewhat pointless assignment to a `const` and then immediate reassignment to `resultDate`.

Reply:
Done